staging/src/k8s.io/client-go/tools/clientcmd/api/v1/types.go

```diff
@@ -51,6 +52,7 @@ type Config struct {
 	Extensions []NamedExtension `json:"extensions,omitempty"`
 }
 
+// Deprecated in v1.34 in favor of kuberc file (see https://kep.k8s.io/3104).
 type Preferences struct {
 	// +optional
 	Colors bool `json:"colors,omitempty"`
```

Comment:
do we not even honor this field in `kubectl`? 🤦 

looks like the only spot we use color is in the warning writer, and that just goes off of terminal autodetection and the NO_COLOR envvar

Reply:
> do we not even honor this field in `kubectl`? 🤦

Nope, it's just not used :disappointed: